altair/utils/schemapi.py

```diff
@@ -99,14 +99,15 @@ def __str__(self):
             return six.text_type(self).encode("utf-8")
 
 
-
```

Comment:
The ground truth for this file is https://github.com/altair-viz/altair/blob/master/tools/schemapi/schemapi.py

Changes to the file should be made there, and then propagated here by running ``tools/generate_schema_wrapper.py``.

Reply:
Have done. However, running `generate_schema_wrapper.py` also generates some diffs to various core.py files, I suspect because upstream has subtly changed the wording in some docs? I've not committed those changes here. Should I?

Reply:
There shouldn't be any diffs, because it's targeted to specific version tags. It might be some small difference in the version of m2r or some other package.